client/components/orderCard.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import {Link} from 'react-router-dom'
+import {AddToCart} from './index'
+
+const OrderCard = props => {
+  return (
+    <li>
+      <Link to={`./orders/${props.order.id}`}>
```

Comment:
Can you delete the `.` in all the `<Link>` paths. The `./` is used for referencing directories and `/` is used for url paths.